osds/verify.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if ! [ -f "./bin/gometalinter" ]; then echo "Download The files";
+	curl -L https://git.io/vp6lP | sh
+fi
+export PATH=${PATH}:${GOPATH}/bin:${GOPATH}/src/github.com/opensds/opensds/osds/bin
+gometalinter --disable-all --enable=gofmt --enable=misspell --enable=golint --exclude=vendor --exclude=test ./...
```

Comment:
misspell may report some changes which may not be required right? Also, it won't fail the build right?

Reply:
+1 , same confusion here, I didn't see any check in CI log files

Reply:
golint output is written to file.. each one we need to analyse and plan during Daito

Reply:
good